chore: add description for presentation requests (#108)
UnumID/UnumID.github.io: docs/api-overview.mdx

  email?: string, // user's email address; optional if phone is provided
  phone?: string, // user's phone number; optional if email is provided
  credentialRequests: CredentialRequest[], // a list of one or more CredentialRequest objects. Encodes which credentials are being asked for; not used if partnerUuid is provided
  partnerUuid?: string, // partner's uuid; optional

>   description?: string // the description that goes to the consent page header, when is empty the default text will be used.

Reviewer: Nearly certain that the `description` attribute was only added to credential requests, not presentation request object.

Reviewer 2: No, this is for the request-level description.

>   description?: string // the description that goes to the consent page header, when is empty the default text will be used.

Reviewer 2: ```suggestion
  description?: string // A description displayed in the consent page header. If no description is provided, a default value will be displayed.
```
